docs/source/algorithms/LoadInstrument-v1.rst

```diff
@@ -9,51 +9,82 @@
 Description
 -----------
 
-Loads an instrument definition file (:ref:`IDF <InstrumentDefinitionFile>`)
-into a workspace, which contains information about detector positions,
-their geometric shape, slit properties, links between values stored in
-ISIS log-files and components of the instrument and so on. For more on
-IDFs see: :ref:`InstrumentDefinitionFile <InstrumentDefinitionFile>`.
+Loads the instrument geometry from either an instrument definition file
+(:ref:`IDF <InstrumentDefinitionFile>`) or a Nexus file into a workspace. The
+geometry contains information about detector positions, their geometric shape,
+slit properties, links between values stored in log-files and components of the
+instrument and so on. For more on IDFs see:
+:ref:`InstrumentDefinitionFile <InstrumentDefinitionFile>`.
 
 By default the algorithm will write a 1:1 map between the spectrum
 number and detector ID. Any custom loading algorithm that calls this as
 a Child Algorithm will therefore get this 1:1 map be default. If the
 custom loader is to write its own map then it is advised to set
 ``RewriteSpectraMap`` to false to avoid extra work.
 
-The instrument to load can be specified by either the InstrumentXML,
-Filename and InstrumentName properties (given here in order of
-precedence if more than one is set). At present, if the InstrumentXML is
-used the InstrumentName property should also be set.
+The instrument to load can be specified by either the ``InstrumentXML`` (IDFs
+only), ``Filename`` and ``InstrumentName`` properties (given here in order of
+precedence if more than one is set). At present, if the ``InstrumentXML`` is
+used the ``InstrumentName`` property should also be set.
+
+The ``Filename`` can either be an absolute or relative path. In the latter case,
+both the ``datasearch.directories`` and instrument directories (accessible via
+``getInstrumentDirectories()`` and ``setInstrumentDirectories()``) are searched for
```

Comment:
On the config service. I think that's a concept that can be linked to in rst.

Reply:
I can't seem to find where the`ConfigService` is in the docs...

Reply:
I thought there might be something here https://github.com/mantidproject/mantid/tree/master/docs/source/concepts I'm mistaken. Ignore the above comment of mine.